.eslintrc.js

```diff
@@ -2,12 +2,14 @@ module.exports = {
 	"extends": "airbnb",
 	"env": {
 		"browser": false,
-		"jest": true
+		"jest": true,
+		"mocha": true,
 	},
 	"plugins": [
 		"import"
 	],
 	"rules": {
+		"global-require": 0,
```

Comment:
Use this eslint rules only for `test` folder. And disable [this](https://eslint.org/docs/rules/func-names) rule for tests too.
```json
"overrides": [{
	"files": ["test/**/*.js"],
	"env": { "mocha": true },
	"rules": { "func-names": 0, "global-require": 0 },
}],
```